lib/rmg/CMapGenerator.cpp

```diff
@@ -62,10 +62,89 @@ CMapGenerator::CMapGenerator(CMapGenOptions& mapGenOptions, int RandomSeed) :
 	zonesTotal(0), tiles(nullptr), prisonsRemaining(0),
     monolithIndex(0)
 {
+	loadConfig();
 	rand.setSeed(this->randomSeed);
 	mapGenOptions.finalize(rand);
 }
 
+void CMapGenerator::loadConfig()
+{
+	static std::map<std::string, ETerrainType> terrainMap
+	{
+		{"dirt", ETerrainType::DIRT},
```

Comment:
For our own map format all terrains have 2 letter codes like dt, gr and so on. VcmiMapFormat class if I am not mistaken. Should we unify somehow the way we represent terrains in json? Or is it ok to have both 2 letter code and long code for the same terrain? Long codes are obviously more readable